### PR TITLE
Blocks id

### PR DESCRIPTION
**Pull request by GroobleDierne**
Use minecraft-data to get blocks id

### CODE REVIEW

File: src/lib/worldGenerations/grass_field.js [hunk 1]
```diff
@@ -2,13 +2,15 @@ const Vec3 = require('vec3').Vec3
...
   const Chunk = require('prismarine-chunk')(version)
+  const mcData = require('minecraft-data')(version)
 
   function generateSimpleChunk () {
     const chunk = new Chunk()
 
     for (let x = 0; x < 16; x++) {
       for (let z = 0; z < 16; z++) {
-        chunk.setBlockType(new Vec3(x, 50, z), 2)
+        chunk.setBlockType(new Vec3(x, 50, z), mcData.blocksByName.grass_block.id)
```
Comment (Karang) [inline src/lib/worldGenerations/grass_field.js:12]: Careful, in old version grass_block is grass

Comment (GroobleDierne) [reply src/lib/worldGenerations/grass_field.js:12]: Do you know which version ?

Comment (Karang) [reply src/lib/worldGenerations/grass_field.js:12]: Bellow 1.13

Comment (GroobleDierne) [reply src/lib/worldGenerations/grass_field.js:12]: Do you recommend to use serv.supportFeature() or a normal condition ?

Comment (rom1504) [reply src/lib/worldGenerations/grass_field.js:12]: serv.supportFeature would be good, I believe we already have something similar in the code

**Comment by GroobleDierne** [comment]
it's good for me.

File: src/lib/worldGenerations/grass_field.js [hunk 2]
```diff
@@ -2,13 +2,16 @@ const Vec3 = require('vec3').Vec3
 
 function generation ({ version }) {
   const Chunk = require('prismarine-chunk')(version)
+  const mcData = require('minecraft-data')(version)
+  const theFlattening = require('../../../app').supportFeature('theFlattening')
```
Comment (rom1504) [inline src/lib/worldGenerations/grass_field.js:6]: this important would start the server, that's not good.
Can you directly important the right thing?

**Review by rom1504 — APPROVED** [review]

**Pull request merged**